integrations/gitlab/gitlab_integration/bootstrap.py

```diff
@@ -60,7 +61,7 @@ def setup_application(
         if use_system_hook:
             gitlab_service.create_system_hook()
         else:
-            webhook_ids = gitlab_service.create_webhooks()
+            webhook_ids = gitlab_service.create_webhooks(groups_paths_for_hooks)
```

Comment:
what happens if user passes multiple tokens? could it be that it will create the webhook multiple times?

Reply:
added validation checks